README.md

```diff
@@ -99,29 +99,30 @@ First, you'll want to have Node.js installed. You can do this by going to [nodej
 
 7. If you used the CLI tool method in step 5 to add yourself to the guest book, **skip this step**. Otherwise:
 
-   Add and commit your changes by running this command:
+   Add and commit your changes by running the following  command:
 
    ```bash
    git commit -am "Your message"
    ```
 
    Change `"Your message"` to your own message. For example, `"Add Alice as a contributor"`.
+Unlike the `git add .` and  `git commit -m "Your message"` approach, the `git commit -am` command skips the staging process. This is because its use of `-a` automatically stage the changes you made via Git.
```

Comment:
Currently, in the live preview, the steps number are unordered --> from 7 to 1. To fix this:
- Add a space before line 109 and indent this line.

Wordings suggestion:
- Let's add a note in a quote so folks can pay attention to it.
- Mention that `git commit -am` is a combination of `-a` and `-m` flags in `git commit`. Folks, especially beginners, usually assume that this command is the combination of `git add` and `git commit` because they are familiar with `git add .`.
- Mention that the `-a` automatically stages all changes in *committed* files.

I have two suggestions. You can choose which one is best to apply here, or maybe you have another idea?

**Suggestion 1**:

```suggestion

   > **Note**: Unlike the `git add .` and `git commit -m "Your message"` approach, in the `git commit -am` command, adding `-a` flag to `git commit` skips the staging process. The `-a` flag automatically stages all your changes in _committed_ files. Combining it with the `-m` flag allows you to write the commit message.  
```

**Suggestion 2**:

```suggestion

   > **Note**: Unlike the `git add .` approach where you add _all_ files (including newly created files), adding the `-a` flag to the `git commit` command automatically stages all your changes in _committed_ files. Combining it with the `-m` flag allows you to write the commit message.
```